p2p/simulations/p2ptest.go

```diff
@@ -64,29 +68,66 @@ func (p *myProtocol) sendMyMessage() {
 
 type Config struct {
 	P2PConfig    p2p.Config
+	StaticNodes  []string
 	RemoteNodeID string // optional. peer node
 	RemoteAddr   string // optional, format 182.87.223.29:39008
 }
 
+func myresolve(id string) (*discovery.Node, error) {
+	nodeHeader := "snode://"
+	id = id[len(nodeHeader):]
+
+	idSplit := strings.Split(id, "@")
+	if len(idSplit) != 2 {
+		return nil, errors.New("invalidNodeError")
```

Comment:
const error defined in the beginning of this file?

Reply:
en, next commit fix this!